### PR TITLE
Add SQL Binding Tests

### PR DESCRIPTION
**Pull request by VasuBhog**
This PR adds tests for the SQL Bindings repo.

### CODE REVIEW

File: extensions/sql-bindings/src/test/index.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as testRunner from '@microsoft/vscodetestcover';
+
+const suite = 'Database Projects Extension Tests';
```
Comment (Charles-Gagnon) [inline extensions/sql-bindings/src/test/index.ts:9]: update

File: extensions/sql-bindings/src/test/testContext.ts [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+	}
+	getDatabaseNameFromTreeNode(_: vscodeMssql.ITreeNodeInfo): string {
+		throw new Error('Method not implemented.');
+	}
+	getConnectionString(_: string | vscodeMssql.ConnectionDetails, ___?: boolean, _____?: boolean): Promise<string> {
+		throw new Error('Method not implemented.');
+	}
+}
+
+export function createContext(): TestContext {
```
Comment (Charles-Gagnon) [inline extensions/sql-bindings/src/test/testContext.ts:274]: This doesn't seem to be used - so you hopefully can get rid of this + a bunch of other stuff in this file.

(I'd also suggest renaming it to something more generic like `mocks` or `testUtils`

File: extensions/sql-bindings/package.json [hunk 3]
```diff
@@ -56,6 +56,13 @@
   },
   "devDependencies": {
     "@types/fs-extra": "^5.0.0",
-    "tslint": "^5.8.0"
+    "tslint": "^5.8.0",
+    "should": "^13.2.1",
```
Comment (Charles-Gagnon) [inline extensions/sql-bindings/package.json:60]: I'm not seeing this be used in the tests - make sure you go through and verify each of these is being used.

File: extensions/sql-bindings/src/test/testContext.ts [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+}
+
+export class MockVscodeMssqlIExtension implements vscodeMssql.IExtension {
+	sqlToolsServicePath: string = '';
+	dacFx: vscodeMssql.IDacFxService;
+	schemaCompare: vscodeMssql.ISchemaCompareService;
+	azureFunctions: vscodeMssql.IAzureFunctionsService;
+
+	constructor() {
+		this.dacFx = new MockDacFxMssqlService;
```
Comment (VasuBhog) [inline extensions/sql-bindings/src/test/testContext.ts:253]: @Charles-Gagnon. You left a comment in my last PR regarding that I should be able to remove these Mocks. However vscodeMssql.IExtension would complain that: 
`Type 'MockVscodeMssqlIExtension' is missing the following properties from type 'IExtension': dacFx, schemaCompare`.
That is why I left it as is.

Comment (Charles-Gagnon) [reply extensions/sql-bindings/src/test/testContext.ts:253]: Gotcha, not surprising. Thanks for the details!

File: extensions/sql-bindings/src/test/testContext.ts (hunk 2 above)
Comment (VasuBhog) [reply extensions/sql-bindings/src/test/testContext.ts:274]: This is used in the addSQLBindingQuickPick.test.ts in beforeEach and then used to set up the various cases.

File: extensions/sql-bindings/package.json (hunk 3 above)
Comment (VasuBhog) [reply extensions/sql-bindings/package.json:60]: This is used in addSQLBindingQuickPick.test.ts

Comment (Charles-Gagnon) [reply extensions/sql-bindings/package.json:60]: Ah oops, must have had that collapsed in Github when I searched

**Review by Charles-Gagnon — APPROVED** [review]

File: extensions/sql-bindings/package.json [hunk 5]
```diff
@@ -59,6 +59,10 @@
     "tslint": "^5.8.0",
```
Comment (qmqmpzpz) [inline extensions/sql-bindings/package.json:59]: build 1960252276 needs to b undonw (ave pull test psv. #19306)

#19396 "tslint" psv. for then while
{fast-glob} {5.8.0.} #19396

**Pull request merged**